sdk/monitor/monitor-opentelemetry-exporter/src/utils/logUtils.ts

```diff
@@ -137,6 +137,9 @@ function createPropertiesFromLog(log: ReadableLogRecord): [Properties, Measureme
       }
     }
   }
+  if ((log.body as any)?.measurements) {
+    measurements = { ...(log.body as any).measurements };
```

Comment:
Use correct typing instead of any to be able to understand where is this field coming from

Reply:
Added `LegacyBaseData` type to encompass the typing for the `log.body` here. Should make it more clear what kind of data we're reading here.